templates/create-daml-app/ui/src/react-app-env.d.ts

```diff
@@ -0,0 +1,4 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+/// <reference types="react-scripts" />
```

Comment:
That file seems useless?

Reply:
No idea what this is

Reply:
This is bringing some typings from react into scope. It's coming from `create-react-app` as well. I would leave it.